src/xcode/ENA/ENA/Source/Services/Risk/CalculationV2/Models/CWARange.swift

```diff
@@ -0,0 +1,61 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+import Foundation
+
+struct CWARange: Decodable {
+
+	// MARK: - Init
+
+	init(from range: SAP_Internal_V2_Range) {
+		self.min = range.min
+		self.max = range.max
+		self.minExclusive = range.minExclusive
+		self.maxExclusive = range.maxExclusive
+	}
+
+	// MARK: - Internal
+
+	let min: Double
+	let max: Double
+
+	let minExclusive: Bool?
+	let maxExclusive: Bool?
```

Comment:
Thats not internal but quite public ;) 
Could you add comments on these properties. Are `minExclusive` and `maxExclusive` set externally?

Reply:
Well, they were indeed `internal` since that is the default visibility in Swift, not `public`. But since they are not written or read externally we changed them to `private` now 🤓👍